parser/toplevel.ml

```diff
@@ -18,3 +18,14 @@ type command =
   | Require  of mident
   | Print    of string
   | Other    of string*term list
+
+
+type is_opaque = bool
```

Comment:
I find this `is_opaque` type alias weird... I think you are trying to name the 3rd argument in the tuple of `Definition`'s constructor.
It would probably be better to simply document the constructors of the `entry` type instead.